Src/BootCamp.Chapter/Lesson8.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson8
+    {
+        /// <summary>
+        /// Takes an array of floats and returns the highest value.
+        /// </summary>
+        /// <param name="array">Array of floats.</param>
+        /// <returns>Highest float in the array.</returns>
+        public static float Max(float[] array)
```

Comment:
Please rename to be a verb.

Reply:
A lot of other programming languages have a "Max" function to get the highest value, even the Math module has the same one that only takes in two parameters instead of an array.